src/Oscoin/Crypto/Blockchain/Block.hs

```diff
@@ -54,25 +68,46 @@ type Depth = Natural
 -- | Block score.
 type Score = Integer
 
+data Unsealed = Unsealed deriving (Generic, Show)
+
+-- NOTE(adn) In the midst of the refactoring for oscoin#368, at some point
+-- GHC started requesting me a 'Serialise Unsealed' constraint in various
+-- places. My intuition is that /in theory/ we should never serialise an
+-- 'Unsealed' block, so we should do another sweep through the different
+-- constraints and try to pinpoint why is that. For now, the path of least
+-- resistence is to simply provide such (dubious) instance.
+instance Serialise Unsealed
```

Comment:
We should maybe turn this into an issue - might be a good idea to implement it with `panic` as the implementation.

Reply:
Totally, thanks for the remainder!

Reply:
⚠️ Will open an issue as soon as we merge (the emoji serves to me as a visual cue).